web/shapeworks/src/views/Data.vue

```diff
@@ -246,8 +299,8 @@ export default defineComponent({
     position: absolute;
     display: flex;
     justify-content: space-between;
-    left: 650px;
-    width: calc(100% - 650px);
+    left: 500px;
+    width: calc(100% - 500px);
```

Comment:
These properties seem pretty fundamental to the layout of the page. I think this is fine for now, but eventually we will need to add new tabs that will also need to duplicate all this CSS. We should think about moving this styling up a layer to keep things DRY